frontend/src/main/components/OurTable.js

```diff
@@ -10,7 +10,7 @@ var tableStyle = {
     "maxWidth": "-moz-fit-content",
     "margin": "0 auto",
     "overflowX": "auto",
-    "whiteSpace": "nowrap"
+    "whiteSpace": "nowrap",
```

Comment:
Does this comma serve a purpose or is this a leftover from previous/undone changes?

Reply:
reverted undone change